app/firebaseApp.js

```diff
@@ -0,0 +1,28 @@
+// Import the functions you need from the SDKs you need
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/auth';
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
```

Comment:
@abhinavbansal19961996 Pls keep it in .env file

Reply:
how to fetch it on client side from .env? @zeenfaizpy

Reply:
@abhinavbansal19961996 Since Nextjs is server rendered, you can keep all your env data in .env file. No need to use in client side.

Reply:
<img width="1256" alt="image" src="https://user-images.githubusercontent.com/18628304/165167510-4aa99899-b906-4939-8709-431dc9b89535.png">

Reply:
@abhinavbansal19961996 see here https://blog.jarrodwatts.com/the-ultimate-guide-to-firebase-with-nextjs

Reply:
> <img alt="image" width="1256" src="https://user-images.githubusercontent.com/18628304/165167510-4aa99899-b906-4939-8709-431dc9b89535.png">

Seems old comment in 2017 ( 5 years before )